pkg/jsx/jsx-runtime.js

```diff
@@ -0,0 +1,16 @@
+import { h, text } from 'hyperapp'
+
+const jsx = (type, { children = [], ...props }) =>
+  typeof type === 'function'
+    ? type(props || [], children)
+    : h(
+        type,
+        props || {},
+        []
+          .concat(children)
+          .map((any) =>
+            typeof any === 'string' || typeof any === 'number' ? text(any) : any
+          )
+      )
+
+export { jsx, jsx as jsxs, jsx as jsxDEV }
```

Comment:
What is `jsxDEV`?

Reply:
I think it's so we can hook in extra stuff for development usage (devtools and what not).

Reply:
Since we're not changing anything in the implementation for jsxDEV, I'll remove it from the exports in this PR. (it needs to be explicitly enabled anyways)